sdks/python/apache_beam/ml/transforms/embeddings/tensorflow_hub_test.py

```diff
@@ -21,7 +21,7 @@
 import apache_beam as beam
 from apache_beam.ml.transforms.base import MLTransform
 
-hub_url = 'https://tfhub.dev/google/LEALLA/LEALLA-small/1'
```

Comment:
I think the upstream model has been updated.  The tests that were passing earlier are failing now. It fails when loading this model. In the meantime, change the model and use the name from the display https://www.tensorflow.org/hub